api/converter/from_pb.go

```diff
@@ -440,6 +442,31 @@ func fromIncrease(pbInc *api.Operation_Increase) (*operation.Increase, error) {
 	), nil
 }
 
+func fromSetIndex(pbInc *api.Operation_SetIndex) (*operation.SetIndex, error) {
+	parentCreatedAt, err := fromTimeTicket(pbInc.ParentCreatedAt)
+	if err != nil {
+		return nil, err
+	}
+	positionAt, err := fromTimeTicket(pbInc.PositionAt)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Since it is the creation time of the target to be changed, how about changing the variable name to `targetCreatedAt`?

Reply:
It's corrected https://github.com/yorkie-team/yorkie/pull/221/commits/70e81a4bee1953c01014b8f110617cd6d7cd0bf6